src/profile-logic/profile-data.js

```diff
@@ -2164,6 +2164,15 @@ export function getThreadProcessDetails(
     }
   }
 
+  if (thread.isPrivateBrowsing) {
+    label += '\nPrivate browsing: yes';
```

Comment:
It would be nice to capitalize this "yes" and "browsing" I think.

We should also probably localize these tooltips, but not for this patch of course.